test/check-application

```diff
@@ -1174,7 +1174,7 @@ class TestApplication(testlib.MachineCase):
         second_tag = "localhost/copybox:latest"
         self.execute(False, f"podman tag localhost:5000/my-busybox {second_tag}")
         # expand details
-        b.click("#containers-images tr:contains('my-busybox') td.pf-v5-c-table__toggle button")
+        b.click(f"#containers-images tr:contains('{second_tag}') td.pf-v5-c-table__toggle button")
```

Comment:
This isn't a "fix" in the sense that the old behaviour was wrong. The original code checks that adding a second tag to an existing image correctly gets picked up by the UI and shown as "additional tag" names in the *original* image info.

You are welcome to *add* code which checks that the new tag also appears as an image by itself (and then also shows `my-busybox` as alternative tag name), but let's please keep that check.

Reply:
I see, my bad. I'll revert this then